ONSdigital/ssdc-rm-docker-dev
# SDCSRM-533 Add support-api to docker-dev

# Motivation and Context

There's a new service to be added

# How to test?



- Build support-api through the PR branch
- Run make up and run some acceptance tests. 
- Once the tests have ran for a couple minutes. Visit the surveys endpoint and you should see all the surveys listed
# Links



[Jira](https://jira.ons.gov.uk/browse/SDCSRM-533)
# Screenshots (if appropriate):
